### PR TITLE
INDG-261 Change structure of microservices list

### PR DESCRIPTION
**Pull request by titkov49**
Changes:
1. Remove microservices name (which was id)
2. Rename Appclass column to Microservices and move to first place
3. Change Launchtime column to DateTime component (previously was common text)
4. Apply monospace font to Hostname column
Screenshot 2021-04-29 at 14 45 58

### CODE REVIEW

**Review by ateska — CHANGES_REQUESTED** [review]
Remarks to naming:

"Microservices List" > "Microservices"
"Microservice" (table header) > "Name"
"Launchtime" > "Launch time"
"Hostname" > "Host"

after these, it is mergeable from my side

**Comment by titkov49** [comment]
@ateska @Pe5h4 @wuujcik 
I've been mocking data for this task and for launchtime I used Date.now() to get timestamp and I found out that DateTime component is for some reason multiplying date by 1000 what makes Date incorrect in this example (April 4 53296 instead of today's date). Is it correct behaviour?

**Comment by ateska** [comment]
1000x multiplier is likely because of second vs.  milliseconds ... the base unit should be **second**

That needs to be corrected to seconds, it is likely leftover from some concept (ElasticSearch sometimes provides miliseconds).

**Comment by titkov49** [comment]
@ateska
Headers and table title has been changed
Screenshot 2021-04-29 at 15 26 08

**Review by ateska — APPROVED** [review]

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Add the changes in asab-config module to ASAB WebUI changelog, please

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,10 +12,13 @@
...
+
 ### Refactoring
 
 - Renaming configuration option FAKE_USERINFO to MOCK_USERINFO and refactoring code accordingly (INDIGO Sprint 210406, [!91](https://github.com/TeskaLabs/asab-webui/pull/91))
 
 - Moving ASAB-Config module navigation under the `Maintenance` tab in the sidebar (INDIGO Sprint 210406, [!92](https://github.com/TeskaLabs/asab-webui/pull/92))
 
 - Refactor userinfo API call to obtain Roles and Resources for active tenant (INDIGO Sprint 210416, [!94](https://github.com/TeskaLabs/asab-webui/pull/94))
+
+- Refactor microservices list (INDIGO Sprint 210430, [!99](https://github.com/TeskaLabs/asab-webui/pull/99))
```
Comment (Pe5h4) [inline CHANGELOG.md:24]: Here should be written what was the actual refactorisation (the purpose of it, what it changed, etc.). This is too vague

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (titkov49) [reply CHANGELOG.md:24]: Got it!

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**